examples/compiled/area.vg.json

```diff
@@ -177,6 +177,7 @@
         }
     ],
     "config": {
+        "boxExtent": 1.5,
```

Comment:
Hmm. `boxExtent` is not a valid Vega config.  In any case, it shouldn't be output in all output Vega specs.